diff --git a/node_modules/deepagents/dist/index.js b/node_modules/deepagents/dist/index.js
index c4ed5a4..c1bac4d 100644
--- a/node_modules/deepagents/dist/index.js
+++ b/node_modules/deepagents/dist/index.js
@@ -15,7 +15,7 @@ import fs$1 from "node:fs";
 import path$1 from "node:path";
 import cp, { spawn } from "node:child_process";
 import fg from "fast-glob";
-import { LangSmithResourceNotFoundError, LangSmithSandboxError, SandboxClient } from "langsmith/experimental/sandbox";
+import { LangSmithResourceNotFoundError, LangSmithSandboxError, SandboxClient } from "langsmith/sandbox";
 import os from "node:os";
 //#region src/backends/utils.ts
 /**
